README.md

```diff
@@ -20,14 +20,34 @@ npm install react-transition-group --save
 yarn add react-transition-group
 ```
 
+## CDN / External
+
+To exclude the react-transition-group package from your primary build and use an external CDN when bundling, link
+to the following CDN:
+
+[https://unpkg.com/react-transition-group/dist/react-transition-group.min.js](https://unpkg.com/react-transition-group/dist/react-transition-group.min.js)
+
```

Comment:
externalizing dependencies is balance between size, usage and geographic location. Libraries like React may make sense to consume from a CDN since it's fairly large, and fairly likely to already be in a user's cache since lots of sites use it. Smaller libraries like this are way less likely to already be cached and are so small I'd argue the latency of a new network request outweighs the benefit of making it external. Not to mention request limit bottlenecks in browsers. 

That said folks should measure and pick for their use cases, but I think generally making all libraries external isn't great default advice and I'd rather just stay out of that game :) to that end we can keep the CDN and dist build info but let's remove the webpack config details since that's more build specific